engine/src/main/scala/cromwell/engine/db/slick/ExecutionComponent.scala

```diff
@@ -161,6 +164,17 @@ trait ExecutionComponent {
       if execution.executionId === executionId
     } yield (execution.status, execution.rc))
 
+  val executionsWithCacheHitWorkflowAndCall = Compiled(
+    (workflowExecutionUuid: Rep[String]) => for {
+      (((e, e1), we1), we) <- executions joinLeft executions on (_.resultsClonedFrom === _.executionId) joinLeft workflowExecutions on {
+        case ((e, maybeE1), we1) => maybeE1.map(_.workflowExecutionId) === we1.workflowExecutionId
+      } joinLeft workflowExecutions on {
+        case (((e, maybeE1), maybeWe1), we) => e.workflowExecutionId === we.workflowExecutionId
```

Comment:
The `we` on 172 is not the same `we` of line 169 and 174 fame, which is a little confusing at first